src/main/java/org/folio/qm/service/impl/ValidationServiceImpl.java

```diff
@@ -97,7 +98,9 @@ public void validateMarcRecord(BaseMarcRecord marcRecord) {
   private boolean containsErrorSeverityType(List<ValidationIssue> validationIssues) {
     return !CollectionUtils.isEmpty(validationIssues) && validationIssues.stream()
       .anyMatch(issue ->
-        issue.getSeverity() != null && issue.getSeverity().equalsIgnoreCase(SeverityType.ERROR.getType()));
+        issue.getSeverity() != null
+          && issue.getSeverity().equalsIgnoreCase(SeverityType.ERROR.getType())
+          && !issue.getTag().matches(TAG_001_REGEX));
```

Comment:
I suppose that the exact error code should be excluded from condition instead of ignoring all 001 ERRORs

Reply:
done